style.css

```diff
@@ -344,3 +344,146 @@ body {
   height: 135px;
 }
 */
+.getStart {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #f9f9fa;
+}
+
+.getStartImg {
+  float: inline-start;
+  margin-left: -15px;
+  width: 111px;
+  height: 74.01807403564453px;
+}
+
+#h2hetstart {
+  width: 249px;
+  height: 50px;
+  align-items: center;
+  font-family: 'Playfair Display', sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1a2236;
+  margin-left: 15px;
+}
+
+.getStartTXT {
+  width: 288px;
+  height: 63px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  margin-left: 15px;
+  color: #1a2236;
+}
+
+.p_input {
+  width: 70px;
+  height: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 10px;
+  font-weight: bold;
+  line-height: 15px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 15px;
+  color: #7d7c84;
+}
+
+#name {
+  width: 288px;
+  height: 52px;
+  border-radius: 4px;
+  margin-left: 15px;
+  border: solid 1px;
+  border-color: #dfdfe0;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1a2236;
+  padding-left: 20px;
+}
+
+#email {
+  width: 288px;
+  height: 52px;
+  border-radius: 4px;
+  border: solid 1px;
+  border-color: #1a2236;
+  background-color: #fff;
+  margin-left: 15px;
+  padding-left: 20px;
+}
+
+#message {
+  width: 288px;
+  height: 112px;
+  padding-left: 20px;
+  border-radius: 4px;
+  border: solid 1px;
+  border-color: #dfdfe0;
+  background-color: #fff;
+  margin-left: 15px;
+}
+
+#getStartbtn {
+  width: 298px;
+  height: 44px;
+  background: #1a2236;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: bolder;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-left: 15px;
+  margin-top: 15px;
+  margin-bottom: 30px;
+  border: none;
+}
+
+.resume {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  width: 318px;
+  height: 137px;
+  background-color: #1a2236;
+}
```

Comment:
- [ ] Kindly remove the margin at the right side of the resume section to follow the Figma design 
### Your design :point_down: 
![Screenshot from 2023-07-18 13-38-13](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/b0887e7d-c4e2-4b61-820f-cd9a0d0d6b00)
### Figma design :point_down: 
![Screenshot from 2023-07-18 13-44-24](https://github.com/amel129/Setup_and_mobile_version_skeleton/assets/91142432/323e7b2a-3528-428f-9881-670c4b95ba14)